beta-src/src/utils/draw-arrow.ts

```diff
@@ -0,0 +1,46 @@
+export default function drawArrow(actionType, source, svgElement, target) {
+  const actionTypeColors = {
+    moveOrder: "#FFFFFF",
+    move: "#000000",
+    moveConvoy: "#2042B8",
+    moveFailed: "#BB0000",
+    moveSupport: "#F8F83D",
+    holdSupport: "#3FC621",
+    retreat: "#BD2894",
+  };
```

Comment:
these should be in the theme or outside of this file in case we want to reuse them